### DISCUSSION TITLE
Is there any way of making ramps/"wavy" ground sections with ogmo editor and setting correct hitboxes?

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2556] [Q&A]
if there's a ramp, character should somewhat smoothly climb it, so hitbox should be either rotated or be smth like triangle form, and as far as i know, HF creates square hitboxes, at least by default...

**Comment by Geokureli** [comment]
Check out the [FlxTilemapExt Demo](https://haxeflixel.com/demos/FlxTilemapExt/). Note that FlxTilemapExt is a flixel-addons feature.

I've never tried this with ogmo, but I assume you would make tiles meant for ramps, place them in ogmo and then tell FlxTilemapExt to treat them with a certain hitbox based on the tile index

**Comment by RichardBray** [comment]
I'm not sure about Ogmo but I've created a video on how to achieve a ramp using Echo if you're interested.

https://www.youtube.com/watch?v=TwHY-yqCOA0&list=PLiKs97d-BatGSvbXmQFCH1QkUzxkRJh5r&index=4&t=2s
